### ISSUE TITLE
(feat): Add script that dumps teacher model labels for fine-tuning datasets

### ISSUE THREAD

**Issue by yair-schiff** [kuleshov-group/dllm-dev#29]
Use either Llama / Flan and save argmax and softmax outputs to disk to be used for knowledge distilation dataset
